calico/release-scripts/tests/test_tags_tars.py

```diff
@@ -29,9 +29,6 @@
     "https://github.com/projectcalico/{component}/archive/{component_version}.tar.gz"
 )
 
-CNI_URL_TEMPL = "https://github.com/projectcalico/{component}/releases/download/{component_version}/{binary}-{arch}"
```

Comment:
Should we not be shipping all the same binaries as we used to?  Just that they're now in this repo instead.

I notice a slack calico user is asking where the calicoctl binary is, for example.

Reply:
> Should we not be shipping all the same binaries as we used to? Just that they're now in this repo instead.

They're all part of the release tgz, which is verified to exist by this suite but doesn't check the contents. We should add a new test for that probably, but it will be an annoying one because it will take some time to download and extract. 

I'll re-add ones for calicoctl, since we decided to ship those on the release directly.